lua/astrocommunity/fuzzy-finder/snacks-picker/init.lua

```diff
@@ -0,0 +1,106 @@
+          desc = "Find lines",
+        }
+        maps.n["<Leader>fa"] = {
+          function() require("snacks").picker.files { cwd = vim.fn.stdpath "config", desc = "Config Files" } end,
+          desc = "Find AstroNvim config files",
+        }
+        maps.n["<Leader>fb"] = { function() require("snacks").picker.buffers() end, desc = "Find buffers" }
+        maps.n["<Leader>fc"] = { function() require("snacks").picker.grep_word() end, desc = "Find word under cursor" }
+        maps.n["<Leader>fC"] = { function() require("snacks").picker.commands() end, desc = "Find commands" }
+        maps.n["<Leader>ff"] = {
+          function()
+            require("snacks").picker.files {
+              hidden = vim.tbl_get((vim.uv or vim.loop).fs_stat ".git" or {}, "type") == "directory",
+            }
+          end,
+          desc = "Find files",
+        }
+        maps.n["<Leader>fF"] = {
+          function() require("snacks").picker.files { hidden = true, ignored = true } end,
+          desc = "Find all files",
+        }
+        maps.n["<Leader>fg"] = { function() require("snacks").picker.git_files() end, desc = "Find git files" }
+        maps.n["<Leader>fh"] = { function() require("snacks").picker.help() end, desc = "Find help" }
+        maps.n["<Leader>fk"] = { function() require("snacks").picker.keymaps() end, desc = "Find keymaps" }
+        maps.n["<Leader>fm"] = { function() require("snacks").picker.man() end, desc = "Find man" }
+        maps.n["<Leader>fo"] = { function() require("snacks").picker.recent() end, desc = "Find old files" }
+        maps.n["<Leader>fr"] = { function() require("snacks").picker.registers() end, desc = "Find registers" }
+        maps.n["<Leader>ft"] = { function() require("snacks").picker.colorschemes() end, desc = "Find themes" }
+        if vim.fn.executable "rg" == 1 then
+          maps.n["<Leader>fw"] = { function() require("snacks").picker.grep() end, desc = "Find words" }
+          maps.n["<Leader>fW"] = {
+            function() require("snacks").picker.grep { hidden = true, ignored = true } end,
+            desc = "Find words in all files",
+          }
+        end
+        maps.n["<Leader>lD"] = { function() require("snacks").picker.diagnostics() end, desc = "Search diagnostics" }
+        maps.n["<Leader>ls"] = { function() require("snacks").picker.lsp_symbols() end, desc = "Search symbols" }
+      end,
+    },
+    {
+      "folke/todo-comments.nvim",
+      optional = true,
+      dependencies = { "folke/snacks.nvim" },
+      specs = {
+        {
+          "AstroNvim/astrocore",
+          opts = {
+            mappings = {
+              n = {
+                ["<Leader>fT"] = function()
+                  if not package.loaded["todo-comments"] then -- make sure to load todo-comments
+                    require("lazy").load { plugins = { "todo-comments.nvim" } }
+                  end
+                  require("snacks").picker.todo_comments()
```

Comment:
This generated an error for me. I looked around and could not figure out how this is supposed to work. I could not find `todo_comments()` in the picker source code.

Reply:
Do you mean it actually had an error, or the Lua language server just lied to you because it doesn't know that the todo-comments plugin adds that function 😂